src/tab/tab.c

```diff
@@ -6,7 +6,7 @@
 /*   By: yoav <yoav@student.42.fr>                  +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/09/14 08:53:02 by yoav              #+#    #+#             */
-/*   Updated: 2022/09/14 09:58:11 by yoav             ###   ########.fr       */
+/*   Updated: 2022/09/17 21:40:49 by alopez-g         ###   ########.fr       */
```

Comment:
redundent please remove

Reply:
I know, any time I enter a file and accidentally save it changes the header, it is sometimes a headache since I dont really know how to go back in a proper way